sockeye/train.py

```diff
@@ -361,7 +363,8 @@ def create_data_iters_and_vocabs(args: argparse.Namespace,
             max_seq_len_source=max_seq_len_source,
             max_seq_len_target=max_seq_len_target,
             bucketing=not args.no_bucketing,
-            bucket_width=args.bucket_width)
+            bucket_width=args.bucket_width,
+            use_length_ratios=args.length_ratio_task_weight > 0.0)
```

Comment:
i would decide all of these based on whether args.length_task is not None

Reply:
Agree, changed